avocado-setup.py

```diff
@@ -335,7 +337,15 @@ def bootstrap(enable_kvm=False):
                   err_str="Failed to create test repo dir. Error: ")
     for repo in TEST_REPOS:
         get_repo(repo, TEST_DIR)
-
+        
+    if len(os.listdir(prescript)):
+        os.makedirs('/etc/avocado/scripts/job/pre.d')
```

Comment:
`/etc/avocado/scripts/job/pre.d` and `/etc/avocado/scripts/job/post.d` can be stored in some global variable and re-used

Reply:
sure